midpoint.csv

```diff
@@ -0,0 +1,3 @@
+,a_new,b
```

Comment:
Somehow this `midpoint.csv` file always makes its way into the PR 😄

Reply:
haha yeah, let's have this file removed, @samukweku :smile: